src/snowflake/connector/auth/_auth.py

```diff
@@ -460,7 +465,7 @@ def _read_temporary_credential(self, host, user, cred_type) -> str | None:
                     "this please install keyring module using the following command : pip install "
                     "snowflake-connector-python[secure-local-storage]"
                 )
-                return
+                return None
```

Comment:
is it mypy requiring to explicitly return None? a single `return` would just return None..

Reply:
I think so, otherwise I made a mistake